docs/Service_Discovery_In_EKS.md

```diff
@@ -0,0 +1,102 @@
+# Service Discovery in EKS
```

Comment:
All this information is very helpful. However, I wonder if we should focus on how people can leverage service discovery of their happy applications instead of the K8s details. I think a lot of our happy customers don't know anything about K8s (and they shouldn't have to). All they want to know if how they can talk to two services they deploy.

Reply:
Makes sense. I assumed that Core DNS would be central to that, but maybe I'm not sufficiently tying it into how to work with it practically. Happy to continue clarifying. These are more or less first passes.